bootcamp/tutorials/integration/haystack/full_text_search_with_milvus_and_haystack.ipynb

```diff
@@ -0,0 +1,602 @@
+{
```

Comment:
<p>like -&gt; likes</p><p> </p><p>Bob likes to swim" (to test stemming, or maybe this doesn't work for the default analyzer?)</p><p> </p><p>Charlie likes white dogs"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1524/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>stemming test failed, I kept it <code>Bob likes swimming</code></p>